src/main/scala/com/horizon/exchangeapi/tables/Schema.scala

```diff
@@ -178,13 +178,19 @@ object SchemaTQ {
         sqlu"alter table servicepolicies add column label character varying not null default ''",
         sqlu"alter table servicepolicies add column description character varying not null default ''"
       )
+      case 41 => DBIO.seq(
+        sqlu"alter table patterns add column secretbinding character varying not null default ''"
+      )
+      case 42 => DBIO.seq(
```

Comment:
Can you compress your database changes into one case (case 41). See `case 40` for an example.